docs/codacy-guardrails/codacy-guardrails-getting-started.md

```diff
@@ -87,6 +92,33 @@ On the left side menu of the Codacy extension, please make sure that MCP server
 
 ### 4. Restart your IDE
 
+## How to install - Prerequisites for Windows (WSL)
+
+### 1. Ensure your machine has `node.js` and `npx` installed.
+
+### 2. Install the [WSL extension for VSCode.](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl)
```

Comment:
should we have first a how to install WSL? we have this link in the extension: https://learn.microsoft.com/en-us/windows/wsl/install

Reply:
Indeed, I've added a step with that link.